webapp/components/survey/LabelsEditor/More/More.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const More = ({ open, onClick }) => (
```

Comment:
this is ButtonToggle or ButtonMore?
add the css here, and rename the class accordingly. e.g. labels-editor-btn-toggle or labels-editor-btn-more